nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/Server.java

```diff
@@ -598,10 +598,9 @@ public void decorate(AdminCommandContext context, final Server instance) throws
                     List<ServerRef> serverRefs = c.getServerRef();
                     Iterator<ServerRef> serverRefIter = serverRefs.iterator();
                     boolean allLBEnabled = false;
-                    while (!allLBEnabled && serverRefIter.hasNext()) {
```

Comment:
Are we sure if this condition is not required? In the while loop the value of allLBEnabled is being modified.